var/spack/repos/builtin/packages/ruby/package.py

```diff
@@ -23,7 +23,7 @@ class Ruby(AutotoolsPackage, NMakePackage):
     tags = ["windows"]
 
     license("Ruby AND BSD-2-Clause AND MIT", checked_by="tgamblin")
-
+    version("3.3.4", sha256="fe6a30f97d54e029768f2ddf4923699c416cdbc3a6e96db3e2d5716c7db96a34")
```

Comment:
@TApplencourt following the common package style, we should keep the line break between the `license()` directive and the `version()` directive.

Reply:
I  think he broke `black` hence my change. (see d4be275 and https://github.com/spack/spack/actions/runs/10014130150/job/27683195517) or I missread the diff?

Reply:
`black` removed trailing whitespace on the empty line. An empty line without spaces on it should be fine.

Reply:
Oh! Thanks!